packages/meditrak-server/src/apiV2/postChanges.js

```diff
@@ -202,7 +202,7 @@ const constructSurveyResponseTranslators = models => ({
   user_email: userEmail => translateUserEmailToIdAndAssessorName(models.user, userEmail),
   entity_code: entityCode => translateEntityCodeToId(models.entity, entityCode),
   survey_code: surveyCode => translateSurveyCodeToId(models.survey, surveyCode),
-  answers: answers => answers.filter(a => a.body !== ''), // remove any empty answers
+  answers: answers => ({ answers: answers.filter(a => a.body !== '') }), // remove any empty answers
```

Comment:
This translator had a bit of an unexpected format, where you need to return the _changed or new fields in the survey response object_ rather than just the updated field. Makes a bit of sense when considering the use cases above (swapping `survey_code` to `survey_id` etc.)